roles/parse_operator_metadata/tasks/main.yml

```diff
@@ -1,3 +1,7 @@
+- name: "Create variables"
+  set_fact:
+    parse_operator_metadata_results: {'result': 'pass', 'msg': ''}
+
```

Comment:
It would be better to set `{'result': 'pass', 'msg': ''}` before rescue on line 172
That is the only place we are 100% sure everything is OK.